src/sql_types.rs

```diff
@@ -249,23 +249,27 @@ impl<'a> ArgsIterator<'a> {
         debug_assert!(num_default_args <= num_total_args);
         let start_idx = num_total_args - num_default_args;
         for i in start_idx..num_total_args {
-            defaults[i] =
-                Self::sql_to_graphql_default(default_strs[i - start_idx], arg_types[i])
+            defaults[i] = Self::sql_to_graphql_default(default_strs[i - start_idx], arg_types[i])
         }
 
         defaults
     }
 
-    fn sql_to_graphql_default(default_str: &str, type_oid: u32) -> Option<String> {
+    fn sql_to_graphql_default(default_str: &str, type_oid: u32) -> Option<(String, bool)> {
         let trimmed = default_str.trim();
+        if trimmed.starts_with("NULL::") {
```

Comment:
are you confident this is always reliable wrt uppercase/lowercase? Will it alway be cast with a `::`?

Reply:
> are you confident this is always reliable wrt uppercase/lowercase?

Yes, Postgres converts all cases into capitals. Added a test for this.

> Will it alway be cast with a ::?

A `<arg> <type> default null` form is always converted into a cast of the form `NULL::<type>`. Other, more complex forms of expressions might not work.

Reply:
Also updated docs with the caveat that only simple `default null` are supported for now.